app/controllers/application_controller.rb

```diff
@@ -1,5 +1,17 @@
 class ApplicationController < ActionController::Base
-  # Prevent CSRF attacks by raising an exception.
-  # For APIs, you may want to use :null_session instead.
+  before_action :configure_permitted_parameters, if: :devise_controller?
+  # before_action :authenticate_admin_user!
```

Comment:
I've seen that :authenticate_admin_user! is used in active admin config. so here you don't need to use this method. so delete str.3. Let's stick to the rule that any commented code haven't be pushed to repo.